src/parser/Expression.ts

```diff
@@ -661,18 +663,33 @@ export class AAMemberExpression extends Expression {
 
 }
 
-export class AALiteralExpression extends Expression {
+export class AALiteralExpression extends Expression implements SymbolContainer {
     constructor(
         readonly elements: Array<AAMemberExpression | CommentStatement>,
         readonly open: Token,
-        readonly close: Token
+        readonly close: Token,
+        readonly functionExpression: FunctionExpression
     ) {
         super();
         this.range = util.createRangeFromPositions(this.open.range.start, this.close.range.end);
+        this.buildSymbolTable();
     }
 
+    readonly symbolTable: SymbolTable = new SymbolTable();
+    readonly memberTable: SymbolTable = new SymbolTable();
+
     public readonly range: Range;
 
+    public buildSymbolTable() {
+        this.symbolTable.clear();
+        this.symbolTable.addSymbol('m', { start: this.open.range.start, end: this.close.range.end }, new ObjectType(this.memberTable));
+        for (const element of this.elements) {
```

Comment:
Should/could it add the `m.x* = ...` found in its function values?

Reply:
yeah ... we need to provide the correct symbol table context for those functions, so when they are parsed they add to "m" automatically... hmm...

Reply:
Maybe we could at least check for `m.*` in sub init() ? That's a common pattern (in my experience) to initialize a bunch of values in `init()` ...